app/src/components/governance/ProposalDescription.vue

```diff
@@ -11,7 +11,6 @@
             "
           >
           {{ proposal.description }}
-          <p class="parameter">New Parameter: {{ proposal.changes }}</p>
           <p v-if="proposal.beneficiary">Beneficiary: {{ proposal.beneficiary.address }}</p>
```

Comment:
delete this as well?

Reply:
We need to display beneficiary for Treasury proposals. This is not contained in the description

Reply:
true, let me check, but it is probably superfluous

Reply:
Done